src/App.js

```diff
@@ -166,4 +166,10 @@ class App extends Component {
     }
 }
 
-export default App;
+const mapStateToProps = (state) => {
```

Comment:
Here you could use just shorthand notation:
```
const mapStateToProps = (state) => ({ data: state.sites });
``` 
It's just a matter of taste, though.